src/Content/ContentStorageHandler.php

```diff
@@ -38,7 +37,7 @@ public function __construct(protected ModelWithContent $model)
 	public function all(): Generator
 	{
 		foreach (Languages::ensure() as $language) {
-			foreach ($this->dynamicVersions() as $versionId) {
+			foreach ([VersionId::published(), VersionId::changes()] as $versionId) {
```

Comment:
`Suggestion`: I think we should still keep this DRY and have something like `VersionId::defaults()`, `ContentStorageHandler::defaultVersions()` instead.

Reply:
I went for `VersionId::list()` Not 100% convinced. But maybe we should just mark it as internal and not worry too much about it. We will need something else sooner or later when we will have revisions anyway.